addons/hr_contract/views/hr_contract_views.xml

```diff
@@ -1,17 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-        <record id="act_hr_employee_2_hr_contract" model="ir.actions.act_window">
```

Comment:
To remove in a migration script (see: util.remove_record)

Reply:
👍